README.md

```diff
@@ -18,166 +18,4 @@ CloudFormation). If you try to use such a Lambda with an API Gateway
-Amazon's runtime environment, and there are multiple ways to ensure
-this.
-
-#### Static linking
-
-IOHK's [`haskell.nix`](https://github.com/input-output-hk/haskell.nix)
-can build static Haskell binaries by cross-compiling against musl
-libc. This is convenient, but consider copyleft implications (of
-`libgmp`; `wai-handler-hal` is BSD-3-Clause) if you are distributing
-the binaries to other people.
-
-#### Dynamic linking
-
-The other option is to compile the executable in an environment with
-packages that match the [Lambda runtime
-environment](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). Some
-ideas:
-
-* Build the executable in an "imitation environment" like the [`lambci/lambda`](https://hub.docker.com/r/lambci/lambda) Docker container; or
-* (Untested) build the executable on an Amazon Linux 2 EC2 instance.
-
-### Docker container images
-
-It is possible to package Lambdas into Docker containers for
-deployment. Amazon provide base containers for custom runtimes in the
-repository
-[`amazon/aws-lambda-provided`](https://hub.docker.com/r/amazon/aws-lambda-provided/). One
-nice feature of these images is that they provide a [runtime interface
-emulator](https://docs.aws.amazon.com/lambda/latest/dg/images-test.html#images-test-AWSbase),
-which they fall back to when not running "for real". This makes it
-possible to directly invoke the lambda and see how it behaves. (This
-is less important here because we can serve our application off
-`warp`, but for writing Lambdas not invoked by an API Gateway Proxy
-Integration, it's handy.)
-
-At the time of writing (2021-04-06), the
-`amazon/aws-lambda-provided:al2` (Amazon Linux 2 tag) is the newest
-base image that Amazon recommends for custom runtimes. You can use a
-`Dockerfile` to build your images, but there are commented `.nix`
-files in `wai-handler-hal-cdk/runtime` that show different ways of
-building containers using `nixpkgs`'
-[`dockerTools.buildImage`](https://nixos.org/manual/nixpkgs/stable/#ssec-pkgs-dockerTools-buildImage):
-
-* `container.nix` builds a container by copying the static bootstrap
-  executable into the `amazon/aws-lambda-provided:al2` image; and
-* `tiny-container.nix` constructs a minimal container from scratch,
-  fetching the emulator from Amazon's GitHub.
-
-## Integrating
-
-Actually calling the Lambda is done with a [Lambda proxy
-integration](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html)
-from an [API Gateway REST
-API](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-rest-api.html).
-
-The simplest possible integration sends every request to the Lambda
-(where the `wai` `Application` can return 404 or whatever if the
-endpoint doesn't match). This is done by mapping the paths `/` and
-`/{proxy+}` for the HTTP method `ANY`. We do this in our CDK example,
-and CDK provides a Construct which encapsulates this pattern, making
-it extremely simple to deploy.
```

Comment:
Preserve this section, but either remove references to our example, or turn them into links?